src/pat/equaliser/equaliser.js

```diff
@@ -67,7 +68,10 @@ define([
             $container.trigger('pat-update', {pattern: 'equaliser'});
         },
 
-        _onEvent: function(event) {
+        _onEvent: function(event, extra) {
+            if (!_.isUndefined(extra) && extra.pattern === 'equaliser') {
```

Comment:
I think we should think about a smarter condition because  this breaks nested masonry things:
![image](https://user-images.githubusercontent.com/1300763/33670768-1cf6611c-daa6-11e7-9a7c-4399c2dfa261.png)

Without this patch it works great:
![image](https://user-images.githubusercontent.com/1300763/33670859-57f19318-daa6-11e7-8187-5868116dbde5.png)